examples/scripts/my_test.py

```diff
@@ -30,8 +30,8 @@
         type: bool
 # Specify this value according to your collection
 # in format of namespace.collection.doc_fragment_name
-extends_documentation_fragment:
-    - my_namespace.my_collection.my_doc_fragment_name
+#extends_documentation_fragment:
+#    - my_namespace.my_collection.my_doc_fragment_name
```

Comment:
Convention is to have a space after `#`:
```suggestion
# extends_documentation_fragment:
#     - my_namespace.my_collection.my_doc_fragment_name
```

Reply:
Wow, thanks for the quick feedback :)
Done. I added the space after the # char